mne_bids/mne_bids.py

```diff
@@ -562,22 +558,8 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         The event id dict used to create a 'trial_type' column in events.tsv
     hpi : None | str
```

Comment:
are we keeping `hpi` in this function call? I think that should be handled in `make_bids_basename`

Reply:
Good catch. I think we can now make use of `raw._init_kwargs` ;-) It's very nifty ...